frontend/src/entities/dataset/dataset.reducer.ts

```diff
@@ -40,9 +40,9 @@ const initialState = {
 
 export const getDatasetsList = createAsyncThunk(
     'dataset/fetch_entity_list',
-    async (params: ServerRequestProps) => {
+    async (params?: ServerRequestProps) => {
```

Comment:
why do we need to make params optional? it seems we always call `getDatasetsList` with projectName passed in unless I am missing something. Either way, for something like listNotebookInstances we do pass in params exactly like we do before your change. I think we should maintain consistency.

```
export const listNotebookInstances = createAsyncThunk(
    'notebook/list_resources',
    async (params: ServerRequestProps) => {
        let requestUrl = '/notebook';
        if (params.projectName) {
            requestUrl = `/project/${params.projectName}/notebooks`;
        }
        return axios.get<PagedResponsePayload<NotebookResourceMetadata>>(
            addPagingParams(requestUrl, params)
        );
    }
);
```

Reply:
This was a holdover from when the parameter was optional. Looks like it doesn't need to be anymore.